ui/app/src/state/reducers/pathNavigatorTree.ts

```diff
@@ -52,14 +52,14 @@ const updatePath = (state, payload) => {
   const chunk = state[id];
   chunk.totalByPath[parentPath] = children.total;
   chunk.childrenByParentPath[parentPath] = [];
-  children.forEach((item) => {
-    chunk.childrenByParentPath[parentPath].push(item.path);
-    chunk.totalByPath[item.path] = item.childrenCount;
-  });
   if (children.levelDescriptor) {
     chunk.childrenByParentPath[parentPath].push(children.levelDescriptor.path);
     chunk.totalByPath[children.levelDescriptor.path] = 0;
   }
+  children.forEach((item) => {
```

Comment:
This was causing the children to be rendered before the levelDescriptor in some scenarios. Already fixed in develop